CONTRIBUTING.md

```diff
@@ -6,7 +6,7 @@ Please take a moment to review this document in order to make the contribution p
 
 ## Core ideas
 
-The repository primarily contains SDK packages in TypeScript generated from the commercetools platform API reference.
+The repository primarily contains SDK packages in TypeScript generated from the Composable Commerce API reference.
```

Comment:
```suggestion
The repository primarily contains SDK packages in TypeScript generated from the Composable Commerce Platform API reference.
```

Reply:
We're trying to remove the "Platform" keyword.
See https://github.com/commercetools/commercetools-docs/pull/2007